### ISSUE TITLE
MP ENSEMBLE

### ISSUE THREAD

**Issue by egolol** [softmatterlab/Braph-2.0-Matlab#1251]
- [x] FUN
- [x] CON

**Comment by annacanal** [comment]
**CON_FUN_MP_WU**, error when showing a **nodal superglobal** measure:

```
Error using matlab.ui.control.Table/set
Invalid parameter/value pair arguments.

Error in PlotPropMeasureEnsembleNodalMultilayer/update (line 868)
		            set(pr.table_value, ...

Error in PPMeasureEnsemble_M/update (line 776)
		    get(pr.p, 'UserData').update()

Error in PlotElement/update (line 1127)
		        pe.get('PR_DICT').getItem(prop).update()

Error in PlotElement/draw (line 1107)
		    pe.update()

Error in GUI/draw (line 1216)
		    pe.draw('Parent', gui.pp)

Error in PPAnalyzeEnsembleMP_ME_DICT/cb_measure_gui (line 1096)
		            pr.f_m{f_count} = GUI('pe', result_measure, 'POSITION', [x y w h], 'CLOSEREQ', false).draw();

Error in PPAnalyzeEnsembleMP_ME_DICT/draw/cb_measure_plot_btn (line 829)
		        pr.cb_measure_gui();
 
Error while evaluating UIControl Callback.
```

**Comment by annacanal** [comment]
Uploading data in CON_FUN_MP analyses is done in 2 steps, first the connectivity of a group and then the functional data from that same group is added. We should add a message when the upload window opens to know what first.
![image](https://user-images.githubusercontent.com/31701548/160134447-31dbbc9d-211e-40d1-ac47-ce4a01d721c9.png)

**Comment by annacanal** [comment]
**CON_FUN_MP_WU**, error when showing a **global superglobal** measure:

```
Error using matlab.ui.control.UIControl/set
Invalid parameter/value pair arguments.

Error in PlotPropMeasureEnsembleGlobalMultilayer/update (line 855)
		            set(pr.edit_value, ...

Error in PPMeasureEnsemble_M/update (line 776)
		    get(pr.p, 'UserData').update()

Error in PlotElement/update (line 1127)
		        pe.get('PR_DICT').getItem(prop).update()

Error in PlotElement/draw (line 1107)
		    pe.update()

Error in GUI/draw (line 1216)
		    pe.draw('Parent', gui.pp)

Error in PPAnalyzeEnsembleMP_ME_DICT/cb_measure_gui (line 1096)
		            pr.f_m{f_count} = GUI('pe', result_measure, 'POSITION', [x y w h], 'CLOSEREQ', false).draw();

Error in PPAnalyzeEnsembleMP_ME_DICT/draw/cb_measure_plot_btn (line 829)
		        pr.cb_measure_gui();
 
Error while evaluating UIControl Callback.
```

**Comment by annacanal** [comment]
**CON_FUN_MP_WU**, error when showing the **comparison result** of  **global superglobal measure**:

```
Unable to perform assignment with 0 elements on the right-hand side.

Error in PPComparisonEnsemble_Diff/update (line 1129)
		            tmp_value = value{D_T*L2-tmp_diff};

Error in PlotElement/update (line 1127)
		        pe.get('PR_DICT').getItem(prop).update()

Error in PlotElement/draw (line 1107)
		    pe.update()

Error in GUI/draw (line 1216)
		    pe.draw('Parent', gui.pp)

Error in PPCompareEnsembleMP_CON_FUN_CPDict_WU/cb_measure_gui (line 970)
		                    pr.f_m{f_count} = GUI('pe', cp, 'POSITION', [x y w h], 'CLOSEREQ', false).draw();

Error in PPCompareEnsembleMP_CON_FUN_CPDict_WU/draw/cb_measure_plot_btn (line 776)
		        pr.cb_measure_gui();
 
Error while evaluating UIControl Callback.
```

**Comment by annacanal** [comment]
When pressing **"PlotBrainView"** of a **nodal unilayer measure**, nothing appears. After a second click on the button, the windows open and this error jumps:

```
Error using Element/set (line 721)
BRAPH2:PlotBrainGraphComparisonEnsemble:WrongInput SYMS_SIZE must have length 1 or (0) length while its length 68.
The value of this PlotBrainGraphComparisonEnsemble has been restored,
so you can keep on working if you are using command line.
Nevertheless, there might be problems, so better you check your code!

Error in PlotBrainGraphComparisonEnsemble/getMCRPanel/update_brain_meas_plot (line 3222)
		                    pl.set('SYMS_SIZE', measure_data_inner);

Error in PlotBrainGraphComparisonEnsemble/getMCRPanel (line 3315)
		    update_brain_meas_plot()

Error in PlotBrainGraphComparisonEnsemble/settings/cb_panel_mcr (line 1258)
		        pl.getMCRPanel();

Error in PlotBrainGraphComparisonEnsemble/settings (line 1272)
		    cb_panel_mcr()

Error in PlotBrainView/settings (line 886)
		    f_settings = pr.bg.settings();

Error in PPComparisonEnsemble_Diff/cb_brain_view_fig (line 1535)
		        f_settings = pbv.settings();

Error in PPComparisonEnsemble_Diff/draw/cb_brainview (line 838)
		            pr.cb_brain_view_fig();
 
Error while evaluating UIControl Callback.

Dot indexing is not supported for variables of this type.

Error in PPComparisonEnsemble_Diff/cb_brain_view_fig (line 1541)
		        gui.cb_bring_to_front()

Error in PPComparisonEnsemble_Diff/draw/cb_brainview (line 838)
		            pr.cb_brain_view_fig();
 
Error while evaluating UIControl Callback.
```

**Comment by annacanal** [comment]
**FUN_MP_Analysis:**
The parameters to perform the analysis should be after Label, like in all other pipelines.
![Screenshot 2022-03-25 at 16 17 21](https://user-images.githubusercontent.com/31701548/160149110-ab86e4cc-8355-4d4b-a162-d795a346c7d7.png)

**Comment by annacanal** [comment]
in Pipeline **Functional Multiplex Analysis BUD,** Error when showing results from a **nodal superglobal** measure:

```
Unrecognized function or variable 'y_'.

Error in PlotEnsembleMPLine/update_plot (line 1592)
		    pr.plotline(pr.get('X'), y_)

Error in PlotEnsembleMPLine/settings/update (line 1268)
		        pr.update_plot();

Error in PlotEnsembleMPLine/settings/layer_popup_selector (line 1381)
		            update();
 
Error while evaluating UIControl Callback.
```

Also here there should not be layer to choose since it is a superglobal result (so one for the whole multiplex)
![Screenshot 2022-03-25 at 16 36 45](https://user-images.githubusercontent.com/31701548/160152926-c768ab61-97f7-4b7e-bf5e-411926044754.png)

**Comment by annacanal** [comment]
in Pipeline Functional Multiplex Analysis BUD, Error when showing results from a global superglobal measure, no graph is shown:
![Screenshot 2022-03-25 at 16 40 34](https://user-images.githubusercontent.com/31701548/160153446-9f963aac-6e65-4dea-b9af-c4b2e24a0dbe.png)

**Comment by egolol** [comment]
Solved all except plot brain graph bugs in #1260
